xml/cap_admin_app_autoscaler.xml

```diff
@@ -220,128 +224,107 @@
   <title>Using the App-AutoScaler Service</title>
 
   <para>
-   Create the Service Broker for App-AutoScaler, replacing
-   <replaceable>example.com</replaceable> with the <literal>DOMAIN</literal>
-   set in your <filename>scf-config-values.yaml</filename> file:
+   Push the application without starting it
+   first:
   </para>
 
-<screen>&prompt.user;SECRET=$(kubectl get pods --namespace scf \
---output jsonpath='{.items[?(.metadata.name=="api-group-0")].spec.containers[?(.name=="api-group")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
-
-&prompt.user;AS_PASSWORD="$(kubectl get secret $SECRET --namespace scf --output jsonpath="{.data['autoscaler-service-broker-password']}" | base64 --decode)"
-
-&prompt.user;cf create-service-broker <replaceable>autoscaler</replaceable> username $AS_PASSWORD https://autoscalerservicebroker.<replaceable>example.com</replaceable>
+<screen>&prompt.user;cf push <replaceable>my_application</replaceable> --no-start
 </screen>
 
   <para>
-   Enable access to the service:
-  </para>
-
-<screen>&prompt.user;cf enable-service-access autoscaler -p autoscaler-free-plan</screen>
-
-  <para>
-   Create a new instance of the App-AutoScaler service:
-  </para>
-
-<screen>&prompt.user;cf create-service autoscaler autoscaler-free-plan <replaceable>service_instance_name</replaceable></screen>
-
-  <para>
-   A name of your choice may be used to replace
-   <literal>service_instance_name</literal>.
+   Attach auto-scaling policy to the application:
```

Comment:
```suggestion
   Attach a autoscaling policy to the application:
```

Reply:
looks like other instance of autoscaling don't use a `-`

Reply:
You are right, thanks for pointing it out!